hayeon/programmers-1/음양-더하기.js

```diff
@@ -0,0 +1,21 @@
+function solution(absolutes, signs) {
+  let result = 0;
+
+  for (let i = 0; i < signs.length; i++) {
+    absolutes[i] = signs[i] ? absolutes[i] : -absolutes[i];
+    result += absolutes[i];
+  }
+  return result;
+}
```

Comment:
- 잘푸셨습니다 :) 👍
- 아래와 같이 `*=` 연산자와 `1`, `-1` 을 활용하는 방법도 참고해보시면 좋을 것 같습니다 !
```suggestion
function solution(absolutes, signs) {
  let result = 0;

  for (let i = 0; i < signs.length; i++) {
    absolutes[i] *= signs[i] ? 1 : -1;
    result += absolutes[i];
  }
  return result;
}
```
- 또한, 아래 풀이는 reduce를 활용한 풀이입니다 🙇🏻‍♂️
```js
function solution(absolutes, signs) {
  return absolutes.reduce((acc, cur, idx) => {
    const sign = signs[idx] ? 1 : -1;
    return (acc += cur * sign);
  }, 0);
}
```

Reply:
와 *= 는 생각지도못한.....!!😮 더 깔끔해보이네요!!!!👍🏻